README.md

```diff
@@ -106,35 +113,45 @@ An example of a configuration file can be found in [test_config.ini](https://git
 
 After processing the configuration file, DaLI reads crypto data from native sources and generates a RP2 input ODS file and a RP2 configuration file in the `output` directory or where specified with the `-o` CLI option.
 
-To try DaLI with the example configuration, download the following files:
-* [test_config.ini](https://github.com/eprbell/dali-rp2/tree/main/config/test_config.ini)
-* [test_manual_in.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_manual_in.csv)
-* [test_manual_intra.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_manual_intra.csv)
-* [test_trezor_alice.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_trezor_alice_btc.csv)
-* [test_trezor_bob.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_trezor_bob_btc.csv)
+## Quickstart
+
+Provided DaLI is installed, to generate RP2 input files, open a terminal window (or PowerShell if on Windows).
+
+Create the directory structure and download the example input files.
+
+```shell
+mkdir -p dali/input
+cd dali/
+wget https://raw.githubusercontent.com/eprbell/dali-rp2/main/config/test_config.ini -P input/
```

Comment:
I like this, however you need to add a sentence about installing wget (which is not available by default on Mac and Windows). Also I would leave the deleted section above (110-114) as an alternative in case wget is not available.